core/src/main/java/de/danoeh/antennapod/core/service/playback/PlaybackVolumeAdaptor.java

```diff
@@ -0,0 +1,50 @@
+package de.danoeh.antennapod.core.service.playback;
+
+import de.danoeh.antennapod.core.feed.FeedMedia;
+import de.danoeh.antennapod.core.feed.FeedPreferences;
+import de.danoeh.antennapod.core.feed.VolumeReductionSetting;
+import de.danoeh.antennapod.core.util.playback.Playable;
+
+class PlaybackVolumeAdaptor {
+
+    public void adaptVolumeIfNecessary(PlaybackServiceMediaPlayer mediaPlayer, String affectedFeedIdentifier, VolumeReductionSetting volumeReductionSetting) {
```

Comment:
Can this be static?

Reply:
I'm hesitant: The part dealing with the `PlaybackServiceMediaPlayer` is not static, which could mean less temptation of leaking this object in a static context. The static methods below this one only handle simple objects (I removed another `static` there, instead).